Api/Connection.php

```diff
@@ -10,7 +10,7 @@ class Connection {
          * @param Array $bodyArray
          * @return void
          */
-        public function post($url, $bodyArray = false)
+        public function post($url, $bodyArray = false, $activity = false)
```

Comment:
lets rename this to `hmac_auth` instead of `activity`

Reply:
Renamed.